proposals/005-skip-approval-process/README.md

```diff
@@ -0,0 +1,51 @@
+# SKIP 005 - SKIP Approval Process
+
+Summary: A process for approving future SKIPs.
+
+Owner: Caleb Schoepp <caleb.schoepp@fermyon.com>
+
+Impacted Projects:
+
+- [x] spin-operator
+- [x] `spin kube` plugin
+- [x] runtime-class-manager
+- [x] containerd-shim-spin
+- [x] Governance
+- [ ] Creates a new project
+
+Created: 30/10/2024
+
+Updated: n/a
+
+## Background
+
+SKIPs were designed as a lightweight way to have discussions about changes that are more complex or affect multiple projects. They have worked well for this purpose, but we never codified how we should reach agreement around approving a SKIP. This proposal seeks to change that.
+
+## Proposal
+
+The following conditions must be met for a SKIP to be considered approved:
+
+- The SKIP has been open for comment for at least two weeks and any objecting comments are resolved.
```

Comment:
"Objecting" is subjective. IMO this should be all comments
```suggestion
- The SKIP has been open for comment for at least two weeks and all comments are resolved.
```

Reply:
Good call. I'll followup with this change.